src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -53,14 +53,28 @@ public void evaluate() throws Throwable {
 
                 injector.injectMembers(this);
 
+                Throwable throwable = null;
                 try {
                     decorateWithRules(base).evaluate();
                 } catch (AssumptionViolatedException e) {
                     System.out.printf("Skipping %s%n", description.getDisplayName());
                     e.printStackTrace();
```

Comment:
Normally it's bad practice to both log and throw, but in this case the method is also called from the world cleaner thread and connection handling thread, both of which have no way to propagate errors, so logging is still useful here.